util/src/main/java/tc/oc/pgm/util/attribute/AttributeInstance.java

```diff
@@ -0,0 +1,70 @@
+package tc.oc.pgm.util.attribute;
+
+import com.google.common.base.Preconditions;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+public class AttributeInstance {
+
+  private final net.minecraft.server.v1_8_R3.AttributeInstance handle;
```

Comment:
We want to move away from importing NMS directly. This looks like a `NMSHacks` worthy thing, using reflection?

Reply:
I copied this from the SportPaper CraftAttributeInstance class. This is class is a wrapper for the NMS AttributeInstance class. Not sure how we'd move this to NMSHacks since we do need to store the NMSAttributeInstance.
We could change the handle type to object and move all the calls to NMShack and just cast the handle back to AttributeInstance in NMSHacks